src/brr.mli

```diff
@@ -3138,6 +3138,12 @@ module Window : sig
 
   (** {1:ops Operations} *)
 
+  val open' : ?name:Jstr.t -> ?features:Jstr.t -> t -> Uri.t -> t option
```

Comment:
I don't think using `Uri.t` here is the right choice. It should be either `Jstr.t` or `Uri.t option`  please double check with the docs of [`window.open`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) that the type allows to specify everything that can.

Reply:
Could you please double check the status of the returned object whether it's really compatible with `Window.t`. The docs mention a [`WindowProxy`](https://html.spec.whatwg.org/multipage/window-object.html#the-windowproxy-exotic-object) object.

Reply:
Changed the `Uri.t` to `Jstr.t`.
I'm able to call `Window` functions with the `WindowProxy` object, so it behaves like a normal window object.